### ISSUE TITLE
[regression] 1.0.2 fails to build when using bundled QtColorWidgets

### ISSUE THREAD

**Issue by Shnatsel** [callaa/Drawpile#322]
I'm trying to make an Ubuntu PPA for Drawpile because the packages GetDeb provides is vastly outdated.

Versions 1.0.0 and 1.0.1.1 build fine for me on Ubuntu 14.04, but 1.0.2 fails to build with the following error:

```
In file included from /tmp/tmp.81qLZIPQQ1/source/obj-x86_64-linux-gnu/src/client/drawpile_automoc.cpp:2:0:
/tmp/tmp.81qLZIPQQ1/source/obj-x86_64-linux-gnu/src/client/moc_color_dialog.cpp:199:10: error: 'ColorWheel' has not been declared
         &ColorWheel::staticMetaObject,
          ^
```

Full build log: http://pastebin.com/EhbUNtZv

**Comment by callaa** [comment]
This is fixed by commit c5e016d. Use the latest stable-1.x version or cherry pick this commit until version 1.0.3 is released.

**Comment by Shnatsel** [comment]
Thanks! Unfortunately the commit turned into a patch fails to apply to 1.0.2 release, so I'll just stick to shipping 1.0.1.1 until the next release. That's still _way_ more recent than what getdeb ships.

JFYI the repository is located at https://code.launchpad.net/~shnatsel/+archive/ubuntu/drawpile
